From cb22710859af8f628b658f9397674fbff2a06c31 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Tue, 26 Nov 2019 11:28:15 +0800
Subject: [PATCH] arm: dts: rk322x: add rockchip,disable-device-link-resume for
 vop

vop iommu handled by vop driver, so ignore the iommu operation
when vop call pm_runtime_get_sync/pm_runtime_put_sync

Change-Id: Id8e21902ea3941e9ccef3e3d1b12efbbdee8e337
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 arch/arm/boot/dts/rk322x.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/rk322x.dtsi b/arch/arm/boot/dts/rk322x.dtsi
index 5a2cbad76244..d254dccb45d3 100644
--- a/arch/arm/boot/dts/rk322x.dtsi
+++ b/arch/arm/boot/dts/rk322x.dtsi
@@ -972,6 +972,7 @@
 		clocks = <&cru ACLK_VOP>, <&cru HCLK_VOP>;
 		clock-names = "aclk", "iface";
 		#iommu-cells = <0>;
+		rockchip,disable-device-link-resume;
 		status = "disabled";
 	};
 
-- 
2.35.3

